code/game/objects/items/tanks/tanks.dm

```diff
@@ -80,21 +84,31 @@
 
 	populate_gas()
 
+	reaction_info = new
+	explosion_info = new
+	//cant lump this in ComponentInitialize() since we need the gasmix to be initialized.
```

Comment:
ComponentInitialize is depreicated regardless. you can remove this comment if you'd like